pkg/lightning/backend/importer.go

```diff
@@ -39,9 +39,14 @@ const (
 )
 
 var (
-	requiredTiDBVersion = *semver.New("2.1.0")
-	requiredPDVersion   = *semver.New("2.1.0")
-	requiredTiKVVersion = *semver.New("2.1.0")
+	// Importer backend is compatible with TiDB [2.1.0, 5.0.0).
+	requiredMinTiDBVersion = *semver.New("2.1.0")
+	requiredMinPDVersion   = *semver.New("2.1.0")
+	requiredMinTiKVVersion = *semver.New("2.1.0")
+	// TODO: bump max versions based on the version define in Makefile.
+	requiredMaxTiDBVersion = *semver.New("5.0.0")
+	requiredMaxPDVersion   = *semver.New("5.0.0")
+	requiredMaxTiKVVersion = *semver.New("5.0.0")
```

Comment:
`5.0.0-rc` or `5.0.0-beta` will pass this test since they are lower than `5.0.0` 🤔

Reply:
Indeed, I will address this issue in another PR.

Reply:
Addressed in https://github.com/pingcap/br/pull/806.